client-side/src/stories/Select/Select.jsx

```diff
@@ -1,46 +1,58 @@
 import * as React from 'react';
+import {  useState } from "react"
 import Box from '@mui/material/Box';
 import InputLabel from '@mui/material/InputLabel';
 import MenuItem from '@mui/material/MenuItem';
 import FormControl from '@mui/material/FormControl';
 import SelectMui from '@mui/material/Select';
-import PropTypes, { shape } from 'prop-types';
+import PropTypes from 'prop-types';
 import './select.scss';
 
-const Select = ({className, options, onChange, title,size,widthOfSelect}) => {
+const Select = ({className, options, onChange, title, size, widthOfSelect}) => {
+  
   return ( 
-     <div className='selectWrapper' >
+    <div className='selectWrapper'>
       <Box>
-       <FormControl  size={size} variant="outlined">
-        <InputLabel  className='input' >{title}</InputLabel>
-        <SelectMui  style={{width:widthOfSelect}} label={title}
-          className={`genericSelect ${className ? `genericSelect ${className}` : ''}`}
-          onChange={onChange}
-        >
-          {options.map((option, index) => (
-            <MenuItem key={index} value={option} >
-              {option.text}
-              {option.icon}
-            </MenuItem>
-          ))}
-        </SelectMui>
-      </FormControl>
-    </Box></div>
+        <FormControl size={size} variant="outlined">
+          <InputLabel className='input'>{title}</InputLabel>
+          <SelectMui style={{width: widthOfSelect}} label={title}
+            className={`genericSelect ${className ? `genericSelect ${className}` : ''}`}
```

Comment:
this condition is not correct, in case that user send a calssName in props you will have the class genericSelect twice.
it should be: className={`genericSelect ${className} `}